<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBSelectBankUi</class>
 <widget class="QDialog" name="QBSelectBankUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Select Bank</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Location</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="bankCodeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2">
         <property name="text">
          <string>SWIFT Code</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="swiftCodeEdit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="locationEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel1_3">
       <property name="text">
        <string>Bank Name</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1_4">
       <property name="text">
        <string>Bank Code</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="bankNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox15">
     <property name="title">
      <string>Matching Banks</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="Q3ListView" name="bankListView">
        <column>
         <property name="text">
          <string>Bank Code</string>
         </property>
         <property name="clickable">
          <bool>true</bool>
         </property>
         <property name="resizable">
          <bool>true</bool>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bank Name</string>
         </property>
         <property name="clickable">
          <bool>true</bool>
         </property>
         <property name="resizable">
          <bool>true</bool>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Location</string>
         </property>
         <property name="clickable">
          <bool>true</bool>
         </property>
         <property name="resizable">
          <bool>true</bool>
         </property>
        </column>
        <column>
         <property name="text">
          <string>BIC</string>
         </property>
         <property name="clickable">
          <bool>true</bool>
         </property>
         <property name="resizable">
          <bool>true</bool>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Online Banking</string>
         </property>
         <property name="clickable">
          <bool>true</bool>
         </property>
         <property name="resizable">
          <bool>true</bool>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="onlineCheckBox">
        <property name="text">
         <string>only banks with online banking</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bankCodeEdit</tabstop>
  <tabstop>swiftCodeEdit</tabstop>
  <tabstop>bankNameEdit</tabstop>
  <tabstop>locationEdit</tabstop>
  <tabstop>bankListView</tabstop>
  <tabstop>onlineCheckBox</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>QBSelectBankUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QBSelectBankUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
